GitUI/SpellChecker/SpellCheckerHelper.cs

```diff
@@ -3,8 +3,6 @@
     internal static class SpellCheckerHelper
     {
         public static bool IsSeparator(this char c)
-        {
-            return !char.IsLetterOrDigit(c) && c != '_';
-        }
+            => !"_+-".Contains(c) && !char.IsLetterOrDigit(c);
```

Comment:
How are the other users of `IsSeparator` affected by excluding '+' and '-' as separators?

Reply:
Used by `WordAtCursorExtractor`. So better keep it as it is.